Gemfile

```diff
@@ -42,7 +42,8 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
I think its a good idea to have such comments whenever we fork gems so its clear to the reader why we are not using upstream in each case.